app/src/main/java/com/findmyrecycling/dto/Product.kt

```diff
@@ -2,7 +2,12 @@ package com.findmyrecycling.dto
 
 import com.google.gson.annotations.SerializedName
 
-data class Product(var product : String,var id : Int = 0, @SerializedName("name")var common : String ) {
+data class Product(
+    var product : String,
+    var id : Int = 0,
+    @SerializedName("name")
+    var common : String,
+    ) {
```

Comment:
As this class has multiple parameters, I separated it on multiple lines for better readability.
https://kotlinlang.org/docs/coding-conventions.html#class-headers